nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/MoveToColdStorageTest.java

```diff
@@ -217,9 +217,9 @@ public void shouldNotRecomputeThumbnail() throws IOException, OperationException
     @Test
     public void shouldNotReplaceColdStorageContent() throws IOException, OperationException {
         DocumentModel documentModel = createFileDocument(session, true);
+        coreFeature.waitForAsyncCompletion();
         moveContentToColdStorage(session, documentModel);
 
-        transactionalFeature.nextTransaction();
```

Comment:
I don't understand this change, createFileDocument is not committing the transaction so why waiting for async completion ? shouldn't we move the nextTransaction before moveContent ?

Reply:
>  shouldn't we move the nextTransaction before moveContent ?

` coreFeature.waitForAsyncCompletion();` does  ` transactionalFeature.nextTransaction();` I used the first one because the naming is more explicit. It waits for the async action are over.

If I don't wait, I'll move the doc I created to cold storage before https://github.com/nuxeo/nuxeo-coldstorage/pull/268/files#diff-d84dac376fd6d97e588d996d242f8ef48911894c49a36b520d658ac99922b6d7R106 is executed which will restore this same document before the rest of the test which will fail.